integration_tests/test_basic.py

```diff
@@ -101,3 +100,55 @@ def test_native_call(cronos):
     )
     receipt = w3.eth.wait_for_transaction_receipt(txhash)
     assert receipt.status == 0, "should fail"
+
+def test_statesync(cronos):
+    ## cronos fixture
+    # Load cronos-devnet.yaml
+    # Spawn pystarport with the yaml, port 26650 (multiple nodes will be created based on `validators`)
+    # Return a Cronos object (Defined in network.py)
+
+    ## do some transactions
+    from_addr = "crc1q04jewhxw4xxu3vlg3rc85240h9q7ns6hglz0g"
+    to_addr = "crc16z0herz998946wr659lr84c8c556da55dc34hh"
+    coins = "10basetcro"
+    node = cronos.node_rpc(0)
+    txhash_1 = cronos.cosmos_cli(0).transfer(from_addr, to_addr, coins)["txhash"]
```

Comment:
Looks good overall, but in real use case, bank transfer of `basetcro` will be disable and I think this has been pretty much tested as it comes from cosmos sdk.

Maybe to be more real, we should try to trigger an ethereum transaction (or ibc transaction) and see if the query could be fetched using json-rpc `eth_getTransactionByHash` after the state sync.

Could be done later as improvement

Reply:
ok, let me see.

Reply:
Please check https://github.com/crypto-org-chain/cronos/pull/203/commits/d8c37b8472356e3120ffc1912fb3980b95d0a307

Reply:
When the `basetcro` will be disabled? @thomas-nguy

Reply:
I can add one IBC transaction if necessary.

Reply:
oh its always disabled on chain. because to transfer cro, we can only use ethereum transfer.

yeah if would be good if you can add another state change like ibc or ethereum tx 
(for ibc, it might be difficult because you need to setup multiple chains)

Reply:
I see, let me check~